### ISSUE TITLE
Could `namespace` support ES7 export extensions?

### ISSUE THREAD

**Issue by gausie** [benmosher/eslint-plugin-import#195]
I have an `Endpoints.js` file that looks like

```js
export Bookings from './Bookings';
export Users from './Users';
export Resorts from './Resorts';
export Rooms from './Rooms';
export Hosts from './Hosts';
export Filters from './Filters';
export Reviews from './Reviews';
```

which uses the Stage-1 proposal export extensions (https://github.com/leebyron/ecmascript-more-export-from).

However, I get a namespaces error if I do `import * as Endpoints from './Endpoints';`

**Comment by benmosher** [comment]
Should work if you're using babel-eslint.

What error is reported?

On Feb 29, 2016, at 12:19, Samuel Gaus <notifications@github.com<mailto:notifications@github.com>> wrote:


I have an Endpoints.js file that looks like

export Bookings from './Bookings';
export Users from './Users';
export Resorts from './Resorts';
export Rooms from './Rooms';
export Hosts from './Hosts';
export Filters from './Filters';
export Reviews from './Reviews';

which uses the Stage-1 proposal export extensions (https://github.com/leebyron/ecmascript-more-export-from).

However, I get a namespaces error if I do import * as Endpoints from './Endpoints';

-

**Comment by benmosher** [comment]
Actually, I wonder if this is a bug with `export [default] from './module'` specifically. I don't think I have tests wrapped around that.

**Comment by benmosher** [comment]
I added tests for what I thought the issue was, but they already pass. Can you provide the specific issue?

**Comment by benmosher** [comment]
closing for now, let me know if you can provide a failing test case.
